src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -118,6 +122,25 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isSwitchViews()) {
```

Comment:
Perhaps you can consider pulling this if-else loop out so that it follows the SLAP principle.

Reply:
Ok!